### ISSUE TITLE
[FR]: It would be good to be able to select any yaml key/attribute as the label

### ISSUE THREAD

**Issue by spiritualgeek** [zsviczian/excalibrain#47]
In my notes an `alias` is not always set, but I would have keys/attributes like the following

```
---
title: A Title
class: classa
state: astate
tags:
- topic/atopic
- activite/anactivity
---

# Relations
parent:: [[aparent]]
child:: [[achild]]
next:: [[nextone]]
```
etc

It would be brilliant if you were able to configure what to use as the label. Even a concatenation e.g., `title & "-" & state`

I'm uncertain if this is already possible, and I'm just missing something?

Thanks

**Comment by spiritualgeek** [comment]
This would be wonderful and prevents me from writing a script to update all my notes :) It also means that the view can be built for what makes sense for a set of notes

**Comment by zsviczian** [comment]
I've implemented this in 0.0.21. Will release it shortly.
It has turned out to be quite a "black belt" functionality. You will need to understand a little programming to get by, but if you have the basic knowledge, I think the solution in 0.0.21 is super flexible and very powerful.  In short, you'll be able to write your own displayName function to override the default.
